src/unistd.h

```diff
@@ -0,0 +1,49 @@
+#ifndef _UNISTD_H
```

Comment:
This code comes from 
http://stackoverflow.com/questions/341817/is-there-a-replacement-for-unistd-h-for-windows-visual-c
I'm not sure how to proceed since it doesn't include a licence